docs/powergate/cli/pow_data_get.md

```diff
@@ -1,13 +1,13 @@
-## pow ffs get
+## pow data get
 
-Get data by cid from ffs
+Get data by cid from the storage profile
```

Comment:
Continuing to read `storage profile` makes me wonder if is the right choice. 
TBH, I don't have other clear candidate/replacement... maybe I need to get used to it.